test/interpreter/function-lcm.spec.ts

```diff
@@ -59,7 +59,7 @@ describe('Function LCM', () => {
   it('coerces to number', () => {
     const engine = HyperFormula.buildFromArray([
       ['=LCM("4",2)'],
-      ['=LCM(B2:C2)', '\'4', 2],
+      ['=LCM(B2:C2)', '4', 2],
```

Comment:
Why a change here? LCM in Excel will return `4` correctly when B2 is `'4`

Reply:
Can you show an example? With double quote it will fail with `#VALUE!`

<img width="236" alt="Screenshot 2021-03-31 at 14 58 43" src="https://user-images.githubusercontent.com/4718998/113147945-996c5a80-9231-11eb-8035-9f36d89b76ec.png">

Reply:
<img width="361" alt="Screenshot 2021-03-31 at 15 09 01" src="https://user-images.githubusercontent.com/40573492/113149291-19df8b00-9233-11eb-86a4-d59682508d92.png">
<img width="357" alt="Screenshot 2021-03-31 at 15 09 09" src="https://user-images.githubusercontent.com/40573492/113149295-1a782180-9233-11eb-9ad4-fa605b40c272.png">

Reply:
Formula bar shows that the value is `''4` not `'4`. Excel will return `#VALUE!` with double quoted text and coerce the single quoted.

<img width="231" alt="Screenshot 2021-04-01 at 12 50 59" src="https://user-images.githubusercontent.com/4718998/113283914-2e7e5a80-92e9-11eb-9240-5c7e75a85142.png">